polaris-tokens/package.json

```
@@ -46,5 +46,10 @@
   },
   "browserslist": [
     "extends @shopify/browserslist-config"
+  ],
+  "files": [
+    "dist",
+    "README.md",
+    "package.json"
```

```suggestion
  "files": [
    "dist"
```

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files

> Certain files are always included, regardless of settings:
> - package.json
> - README
> - LICENSE / LICENCE
> - The file in the "main" field